app/sanitizers/rendered_markdown_scrubber.rb

```diff
@@ -3,14 +3,14 @@ def initialize
     super
 
     self.tags = %w[
-      a abbr add aside b blockquote br button center cite code col colgroup dd del dl dt em em figcaption
-      h1 h2 h3 h4 h5 h6 hr i img kbd li mark ol p pre q rp rt ruby small source span strong sub sup table
+      a abbr add b blockquote br button center cite code col colgroup dd del dl dt em figcaption
+      h1 h2 h3 h4 h5 h6 hr img kbd li mark ol p pre q rp rt ruby small source span strong sub sup table
       tbody td tfoot th thead time tr u ul video
```

Comment:
`<aside>` is not used at all since we don't have any relevant styling for it, and users can't add their own styling.

`<em>` was in this array twice.

`<i>` is not used by our markdown parser; instead `<em>` is. If people want to italicize text they can use markdown.